src/main/java/com/google/cloud/tools/ide/login/GoogleLoginState.java

```diff
@@ -352,7 +235,7 @@ public boolean logOut() {
    *         if the user chose not to log out
    */
   public boolean logOut(boolean showPrompt) {
```

Comment:
It doesn't make sense to have a logout for `AccountRoaster`, but I think logout makes sense in `GoogleLoginState`, much like it has `logIn()`. `logOut()` does a few more things like clearing the datastore, notifying UI and listeners, etc.